extension/src/ALObject/ALParser.ts

```diff
@@ -335,7 +336,11 @@ function matchALControl(
       }
       break;
     case "value":
-      control = new ALControl(ALControlType.value, alControlResult[2]);
+      control = new ALEnumValue(
+        ALControlType.enumValue,
+        (alControlResult[2] as unknown) as number,
```

Comment:
Well that double cast is interesting. How come? :o

Reply:
If trying to cast directly from string to number I get this warning:
`Conversion of type 'string' to type 'number' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.`

So I just obeyed... 😜

Reply:
ah, well obeying has gotten us this far :D